cmake/Configure/ConfigureExecutable.cmake

```diff
@@ -7,21 +7,5 @@ function(configure_executable EXECUTABLE_NAME)
     target_include_directories(${EXECUTABLE_NAME} PRIVATE ${PROJECT_INCLUDE_FOLDER})
     target_link_libraries(${EXECUTABLE_NAME} PUBLIC ${LIBRARY_NAME})
 
-    # Include boost
-    target_include_directories(${EXECUTABLE_NAME} PUBLIC ${Boost_INCLUDE_DIRS})
```

Comment:
So, We don't have to link these things to the executable because they are already included in the gqcp library itself?

Reply:
Yes, this is related to the PUBLIC cmake keyword.